triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/TriggerMultiProcessSimulation.scala

```diff
@@ -41,6 +33,9 @@ abstract class TriggerMultiProcessSimulation extends AsyncWordSpec with Abstract
   private val simulationTerminatedNormally = Promise[Unit]()
 
   "Multi process trigger simulation" in {
+    val simulation: ActorSystem[Message] =
+      ActorSystem(triggerMultiProcessSimulationWithTimeout, "simulation")
```

Comment:
Defining the simulation as part of the test now means actor system exceptions will be observed by scalatest